connectivity/mqtt/mqtt_common.c

```diff
@@ -198,8 +198,9 @@ status_t cfg_add_topic(mosq_config_t *cfg, client_type_t client_type, char *topi
 
   if (client_type == client_pub || client_type == client_duplex) {
     cfg->pub_config->topic = strdup(topic);
-  } else if (client_type == client_duplex) {
-    cfg->pub_config->response_topic = strdup(topic);
+    if (client_type == client_duplex) {
+      cfg->pub_config->response_topic = strdup(topic);
```

Comment:
So, can you ensure `response_topic` will be released later?

Reply:
We release it here https://github.com/DLTcollab/tangle-accelerator/pull/614/files#diff-35e03cef68b021838691d01e160d69a0R128